### ISSUE TITLE
login issue with new added user

### ISSUE THREAD

**Issue by lalitgawde** [keikaavousi/fake-store-api#118]
Hi @VictorConstantin98  @malisara
You should just pass the exact users of fakestoreapi.com/users. {"username":"Victor98","password":"parola1"} is not a fakestoreapi sample users. in the documentation you can see this: "You can use any of the users' username and password available in users API to get token. any other usernames return error."

_Originally posted by @keikaavousi in https://github.com/keikaavousi/fake-store-api/issues/97#issuecomment-1653025882_
            Then how to add user and then login with that user.

**Comment by 7k997** [comment]
You cannot as this API is meant to be used to help you with working / previewing your project with real life kind of data. The users are preset and cannot be altered. The main purpose of users is you have access to environment which returns you jwt / token / refresh token which you can then use in application for authorization purposes.

TLDR: Users / products & anything else **cannot** be altered / added / modified -> only fetched :)
